arches_lingo/src/arches_lingo/api.ts

```diff
@@ -79,3 +79,16 @@ export const fetchSearchResults = async (
         throw new Error((error as Error).message || response.statusText);
     }
 };
+
+export const fetchConcepts = async () => {
```

Comment:
Nice. I think the original reason for defending against `response.json()` failing is because invalid routes or 404s and whatnot were responding with HTML, but we've since cleaned that up in https://github.com/archesproject/arches/pull/11167, so I'm happy to start adjusting these to be simpler ⭐